charts/mageai/templates/tests/test-connection.yaml

```diff
@@ -12,4 +12,4 @@ spec:
       image: busybox
       command: ['wget']
       args: ['{{ include "mageai.fullname" . }}:{{ .Values.service.port }}']
-  restartPolicy: Never
+  restartPolicy: OnFailure
```

Comment:
This is for [chart test](https://helm.sh/docs/topics/chart_tests/). So I'm not sure what failure you want to avoid here.